billing/v1/billing.proto

```diff
@@ -431,7 +431,7 @@ service Billing {
   }
 
   // WORK-IN-PROGRESS: Returns all registered customfields
-  rpc ListCustomField(google.protobuf.Empty) returns (CustomFieldList) {
+  rpc ListCustomField(google.protobuf.Empty) returns (stream CustomField) {
     option (google.api.http) = {
       get: "/v1/customfieldlist"
```

Comment:
For List rpc, add `POST` to the method and `:read` at the end
```
post: "/v1/customfield:read"
```

Reply:
Applied